mongodbatlas/resource_mongodbatlas_database_user.go

```diff
@@ -109,18 +109,15 @@ func resourceMongoDBAtlasDatabaseUser() *schema.Resource {
 			"scopes": {
 				Type:     schema.TypeSet,
 				Optional: true,
-				Computed: true,
```

Comment:
This is more a question than a comment on the review but @marinsalinas  or @coderGo93 we likely need a ticket to correct this for roles as well as I have a feeling this bug will impact them as well as they function similar here.  Correct?

Reply:
If the api doesn't have "default" values for compute and it doesn't detect changes then the solution would be probably the same as this PR.

Reply:
Ah, actually just realize the issue - roles is set as optional and it's required: https://docs.atlas.mongodb.com/reference/api/database-users-create-a-user  
Another (not related but noticed in review) bug, yes?

Reply:
You're right, it is required the parameter roles, not sure why is optional, it should be changed to required and test it again to sure if the bug still happening or not.

Reply:
I'll make a ticket for that and then close this comment in a bit then.  Thanks

Reply:
INTMDB-147 created.